assets/script.js

```diff
@@ -196,13 +200,17 @@ function getArtistInfo(userInput) {
             } else {
               // Appending the album
               $("#albums>ul").append(
-                '<li><a id="albums" class="waves-effect waves-light modal-trigger" href="#album-modal"><img src="' +
+                '<li><a id="albums" class="image waves-effect waves-light modal-trigger" href="#album-modal"><img class="materialboxed" src="' +
                   albumImage +
-                  'alt="' +
+                  '" alt="' +
                   albumName +
-                  "/></a>" +
+                  '" title=" ' + albumName + ' by ' + response.topalbums["@attr"].artist +
+                  '"/></a' +
                   "</li>"
               );
+              $("a.image").each(function () {
+                $(this).attr("title", $(this).find("img").attr("title"));
```

Comment:
Why not just apply this attribute as part of the append, this is wasting resources